packages/block-library/src/navigation-menu/index.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Server-side rendering of the `core/navigation-menu` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/navigation-menu` block on server.
+ *
+ * @param array $attributes The block attributes.
+ * @param array $content The saved content.
+ * @param array $block The parsed block.
+ *
+ * @return string Returns the post content with the legacy widget added.
+ */
+function render_block_navigation_menu( $attributes, $content, $block ) {
+	return prepare_navigation( $block );
+}
+
+/**
+ * Walks the inner block structure and retunrs an HTML list of it.
+ *
+ * @param array $block The block.
+ *
+ * @return string Returns  an HTML list from innerBlocks.
+ */
+function prepare_navigation( $block ) {
+	$block_tree = '';
```

Comment:
Unsure about the name `$block_tree`. It's not a tree, it's a string. How about `$html`?

Reply:
another leftover :)